src/kibana/components/agg_response/geo_json/geo_json.js

```diff
@@ -25,6 +25,21 @@ define(function (require) {
       });
 
       var chart = {};
+
+      chart.tooltipFormatter = function (feature) {
```

Comment:
Move this function to its own file like the point_series and hierarchical `tooltipFormatter` 

See `src/kibana/components/agg_response/point_series/_tooltip_formatter.js`